docs/Benchmarks.md

```diff
@@ -0,0 +1,94 @@
+# Benchmarks
+
+This document explains how to benchmark ASP.NET Core changes. It is intended for Microsoft employees.

```

Comment:
I don't see anything about microbenchmarks 
1. the ones checked into dotnet/aspnetcore that are never run automatically
2. the ones in dotnet/performance -- do any of those run through code that exists in dotnet/aspnetcore

If crank supports any of these, and I hear it now supports dotnet/performance, we should link to instructions.
for the ones it doesn't support, we should at least point out they exist, and ideally link to instructions 

this is where the commentary on how to choose relevant benchmarks might come in handy. let's try to couch these instructions in terms that a new hire can use, who doesn't know about any of these things.

Reply:
particularly since this doc is titled "how to benchmark changes" not "how to do system level benchmark of changes"

Reply:
The micro benchmarks in this repository were made mostly for investigations and not regression detections.